src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -51,22 +53,22 @@ public static IEnumerable<string> GetSearchTags(this FunctionDescriptor member)
 
         #region Overloads specifying XDocument
 
-        public static string GetDescription(this TypedParameter parameter, XDocument xml)
+        public static string GetDescription(this TypedParameter parameter, XmlReader xml)
```

Comment:
I have recommended `XmlReader xml` parameter to be removed in other comments below. If these three methods become too simple/empty, then we may want to move the logic of calling `GetMemberElement` directly into `FunctionDescriptor` and get rid of these three methods.